rust/kernel/cpumask.rs

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Cpumask variables and related functions.
+//!
+//! C header: [`include/linux/cpumask.h`](../../../../include/linux/cpumask.h).
+
+use crate::bindings;
+use core::iter::Iterator;
+
+/// A struct for returning online cpu index.
+///
+/// It has a cpu index. This index changes to the next online cpu index in each iteration. The last
+/// index value is None indicating there is no valid online cpu index.
+pub struct CpuOnlineIndex {
```

Comment:
Maybe adding `Iter` makes the name a bit more clear, e.g. `OnlineCpusIndexIter` would also match the suggested docs above.